config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml

```diff
@@ -265,6 +265,65 @@ presubmits:
             cpu: 4
             memory: 6Gi
 
+  - name: pull-containerd-node-e2e-systemd-cgroup-1-6
+    always_run: false
+    optional: true
+    cluster: k8s-infra-prow-build
+    max_concurrency: 8
+    decorate: true
+    branches:
+    - release/1.6
+    decoration_config:
+      timeout: 100m
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+    annotations:
+      testgrid-dashboards: sig-node-containerd,containerd-presubmits
+      testgrid-tab-name: pull-containerd-node-e2e-1-6-systemd-cgroup
```

Comment:
Its better to keep the same name for the  job and testgrid tab

Reply:
Good catch, done.